config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -4,14 +4,16 @@ presubmits:
     always_run: true
     skip_report: false
     max_concurrency: 12
-    branches:
-    - master
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
       preset-bazel-scratch-dir: "true"
       preset-e2e-scalability-common: "true"
       preset-e2e-scalability-presubmits: "true"
+    annotations:
+      fork-per-release: "true"
```

Comment:
don't we need "testgrid-create-test-group" ?

Reply:
Do we? There is already a group:
https://k8s-testgrid.appspot.com/presubmits-kubernetes-scalability#pull-kubernetes-e2e-gce-100-performance
I do not see any help about that option.

Reply:
I'm not sure - we have it in kubemark-big. I think those two should be consistent (not sure which way though)

Reply:
There are other presubmits with all combinations of 0/1/2 of those options present.

Reply:
That line was added recently and independently:
https://github.com/kubernetes/test-infra/commit/994cf9001c22dd8cc61c000a9260498d69273489

Reply:
https://github.com/kubernetes/test-infra/pull/15023#issuecomment-547669836 - that supports adding the flag. Done.